src/game_launcher.cpp

```diff
@@ -925,14 +926,14 @@ bool game_launcher::play_multiplayer()
 		gui2::show_error_message(video(), _("Error while playing the game: ") + e.message);
 	} catch (mapgen_exception& e) {
 		gui2::show_error_message(video(), std::string(_("Map generator error: ") + e.message));
-	} catch(network::error& e) {
+	} catch(wesnothd_error& e) {
 		if(e.message != "") {
-			ERR_NET << "caught network::error: " << e.message << std::endl;
+			ERR_NET << "caught network error: " << e.message << std::endl;
 			gui2::show_transient_message(video()
 					, ""
 					, translation::gettext(e.message.c_str()));
 		} else {
-			ERR_NET << "caught network::error" << std::endl;
+			ERR_NET << "caught network error" << std::endl;
```

Comment:
Not sure what you mean, i didn't change when execptions are caught, the referenced line just changes a string.